src/snowflake/cli/_plugins/git/commands.py

```diff
@@ -98,6 +98,21 @@ def _validate_origin_url(url: str) -> None:
         raise ClickException("Url address should start with 'https'")
 
 
+def _unique_new_object_name(
+    om: ObjectManager, object_type: ObjectType, proposed_fqn: FQN
+) -> str:
+    result = (
+        FQN.from_string(proposed_fqn.name)
+        .set_schema(proposed_fqn.schema)
+        .set_database(proposed_fqn.database)
+    )
+    i = 1
+    while om.object_exists(object_type=object_type.value.cli_name, fqn=result):
+        result.set_name(proposed_fqn.name + str(i))
+        i += 1
+    return result.name
```

Comment:
Instead of data - append uuid? Alternatively list existing integrations instead of checking for each one

Reply:
I've changed that to `list existing object` to get them in one query.
I prefer to stay with dummy algorithm so the generated names will be more readable for users